FAQ.md

```diff
@@ -79,3 +79,10 @@ from the API groups that it needs (be they Kubernetes types or your own).
 See the [scheme builder
 docs](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/scheme) for
 more information.
+
+### Q: When using envtest with a third party API, why do I see the error: `no matches for kind "Kind" in version "<group>/<version>"`?
+
+**A**: While testing a third party API with `envtest`, in addition to registering the API type with the test client, you would also
+have to register it with the test server. To do so, download the associated CustomResourceDefinition's manifest locally
+and add its path to [`Environment.CRDDirectoryPaths`](https://pkg.go.dev/github.com/kubernetes-sigs/controller-runtime/pkg/envtest#Environment).
```

Comment:
The solution in the issue is to use the go module : `	filepath.Join(build.Default.GOPATH, "pkg", "mod", "github.com", "owner", "theirs@version", "config", "crd", "bases"),`

I do not think that we should tell fr the users add a CRD manually in the config/crds which is the place where it will be in kubebuilder/sdk because: 
- users will apply that with the make targets when should not
- sdk features to generate a bundle will add the crd as well. so, would it be really desirable?  

Also, could we add here an example of a valid config? wdyt?

Reply:
>The solution in the issue is to use the go module

That path is meant to be opaque, i.e. shouldn't be recommended to users. Side note: this is no different than downloading a CRD locally using `curl https://my.domain/crd.yaml`.

Reply:
Hi @estroz, 

I do not think we are addressing it properly yet. It is a very common question and I think it can be better covered.  

- IMO: It is missing an explanation where describe that EnvTest requires to have access to the CRD of the external-types schemas added and used. Then, we need to clarify that has a few options to do that. 
- Also, I think it is missing an example by point out from the go mod. 

Then, see that the kubebuilder scaffold is: 

```go
By("bootstrapping test environment")
testEnv = &envtest.Environment{
	CRDDirectoryPaths:     []string{filepath.Join("..", "config", "crd", "bases")},
	ErrorIfCRDPathMissing: true,
}
```

So, users will read this explanation and add the manifest manually in the config/crd dir as you suggested. So, I have a few questions: Could not it bring side effects for the project/tools helpers? In sdk, will not the `make bundle` use the CRD added there to generate the bundle? In KB/SDK, will not the makefile targets start to install the CRDs. Should we really suggest that? Have we other options? Should we make a small note here and describe that if you are using sdk/kb then, in order to avoid side effects you might prefer to use the go option or create a dir to copy all crds for there and add the external ones as well?

Reply:
Sounds like you're asking for the envtest docs to be improved, not the FAQ response.

Reply:
I hope that I could clarify all my thoughts in ; https://github.com/kubernetes-sigs/controller-runtime/pull/1393#pullrequestreview-604042047. So, I am closing this one to avoid misunderstands and make it easier for others to follow up on this one.